projects/ngx-speculoos/src/lib/matchers.jasmine.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  toBeChecked,
+  toContainText,
+  toHaveClass,
+  toHaveSelectedIndex,
+  toHaveSelectedLabel,
+  toHaveSelectedValue,
+  toHaveText,
+  toHaveValue,
+  MatcherFunction
+} from './matchers';
+
+const jasminify = (v: MatcherFunction) => {
+  return (_: jasmine.MatchersUtil, _: Array<jasmine.CustomEqualityTester>): jasmine.CustomMatcher => ({
```

Comment:
This doesn't compile:

> ERROR in projects/ngx-speculoos/src/lib/matchers.jasmine.ts(14,11): error TS2300: Duplicate identifier '_'

Reply:
😅I have done this modification with the CLI running the test and these error doesn't pop up...